mne/preprocessing/ica.py

```diff
@@ -403,12 +406,16 @@ def _fit_epochs(self, epochs, picks, decim, verbose):
             self.max_pca_components = len(picks)
             logger.info('Inferring max_pca_components from picks.')
 
+        # this should be a copy (picks a list of int)
         data = epochs.get_data()[:, picks]
+        # this will be a view
         if decim is not None:
-            data = data[:, :, ::decim].copy()
+            data = data[:, :, ::decim]
```

Comment:
Not sure. There will be a definite memory cost (and small speed cost) to doing a memcopy here. It is true there might be speed losses down the line due to loss of SIMD instructions. Would it suffice to time running ICA on raw with `decim=2` before and after the change?